packages/react-core/src/demos/HelperText.md

```diff
@@ -0,0 +1,200 @@
+---
+id: Helper text
+section: components
+---
+
+import MinusIcon from '@patternfly/react-icons/dist/esm/icons/minus-icon';
+import ExclamationTriangleIcon from '@patternfly/react-icons/dist/esm/icons/exclamation-triangle-icon';
+import CheckCircleIcon from '@patternfly/react-icons/dist/esm/icons/check-circle-icon';
+import ExclamationCircleIcon from '@patternfly/react-icons/dist/esm/icons/exclamation-circle-icon';
+import CheckIcon from '@patternfly/react-icons/dist/esm/icons/check-icon';
+import TimesIcon from '@patternfly/react-icons/dist/esm/icons/times-icon';
+
+## Demos
+
+### Static variant with static text
+
+In this demo, the static variant of the helper text item component (the default) is used, and the text itself will always be visible to users and will never change.
+
+The `aria-describedby` attribute is passed into the text input component and is linked to the `id` of the helper text component. This allows assistive technologies to notify users of the helper text content when the input receives focus, which can be helpful if a user navigates away from and then back to the input.
+
+Note that this demo does not validate the text input component. When it would need to be validated, there are other steps that would need to be taken to make it accessible, such as passing in `aria-invalid` and `aria-live` attributes to the appropriate components.
+
+```ts
+import React from 'react';
```

Comment:
Is there a particular reason that you built the demos inside the .md rather than putting them in standalone files in `demos/examples/<component>/<demo name>.tsx`?

Reply:
The component demo I was using as reference was set up that way so I think I was just following that as I was building things out. Separate demos have now been created, though. Good call out!